library/src/components/Extensions.tsx

```diff
@@ -1,34 +1,112 @@
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 
-import React from 'react';
+import React, { useContext, useState } from 'react';
```

Comment:
See my previous comment about context.

```suggestion
import React, { useState } from 'react';
```